public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Edustripe Tech</title>
+</head>
+<body>
+  <div id='root'></div>
+</body>
+</html>
```

Comment:
I love how clean this HTML file is. Really great job buddy.

Reply:
+1 💪🏾 can we have another `meta` tag here for author and description? That way our site could be a bit more friendlier with for SEO something like this would do
```
  <meta name="description" content="Edustripe School Management System">
  <meta name="author" content="Edustripe">
```

Reply:
> +1 💪🏾 can we have another `meta` tag here for author and description? That way our site could be a bit more friendlier with for SEO something like this would do
> 
> ```
>   <meta name="description" content="Edustripe School Management System">
>   <meta name="author" content="Edustripe">
> ```

Thanks @tolulope-od . Done